spec/acceptance/mod_php_spec.rb

```diff
@@ -44,7 +44,7 @@ class { 'apache::mod::php': }
       describe file("#{apache_hash['mod_dir']}/php7.4.conf") do
         it { is_expected.to contain 'DirectoryIndex index.php' }
       end
-    elsif os[:family] == 'redhat' && os[:release] =~ %r{^8\.}
+    elsif os[:family] == 'redhat' && os[:release] =~ %r{^8($|\.)}
```

Comment:
looks a bit complicated. `%r{^8}` should be fine?

Reply:
Quite agree, at the current rate of RedHat releases I think we can ignore the issues that might arise the release following RedHat 79 :grin: …

Reply:
`/^8\b/` should also do it as expected